docker/README.md

```diff
@@ -0,0 +1,40 @@
+# Docker Image
+
+## Deps
+
+- [Docker](https://docs.docker.com/engine/install/)
+- [Docker Compose](https://docs.docker.com/compose/install/) For ease of using locally.
+
+## Setup
+
+See [Local Development Setup](../SETUP.md) instructions.
+
+## Build
+
+From this (`docker`) directory
+
+### docker
+
+```sh
+docker build .. -f ./Dockerfile
+``` 
+
+### docker-compose
+
+```sh
+docker-compose build hubble
+```
+
+## Run
+
+From this (`docker`) directory:
+```sh
+mkdir -p testData/dev-chain
+cp ../genesis.json ./testData/genesis.json
```

Comment:
IIUC, if this is the first time the user clones the repo and decides to run docker, they might not have `genesis.json` as this one is generated on contract deployment.

Reply:
I have the `Setup` section above which should get them there, but now realize those instructions could be more verbose/explicit and docker specific. Will update.